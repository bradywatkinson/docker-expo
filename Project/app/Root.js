import React from 'react';

import loadAssets from 'app/utilities/loadAssets';

import AppWrapper from 'app/components/AppWrapper';
import RootNavigation from 'app/navigation/RootNavigation';


const RootComponent = () => (
  <AppWrapper>
    <RootNavigation />
  </AppWrapper>
);

export default loadAssets(RootComponent);
