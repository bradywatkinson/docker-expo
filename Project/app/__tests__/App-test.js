import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';

import Root from '../Root';

describe('<Root />', async () => {
  it('renders the loading screen', async () => {
    const tree = shallow(<Root />);
    expect(tree).toMatchSnapshot();
  });

  it('renders the root without loading screen', async () => {
    const tree = shallow(<Root skipLoadingScreen />);
    expect(tree).toMatchSnapshot();
  });
});
