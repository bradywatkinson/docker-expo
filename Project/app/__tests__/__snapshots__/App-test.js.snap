// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> renders the loading screen 1`] = `
<AppLoading
  onError={[Function]}
  onFinish={[Function]}
  startAsync={[Function]}
/>
`;

exports[`<Root /> renders the root without loading screen 1`] = `<RootComponent />`;
